package.json

```diff
@@ -45,6 +45,7 @@
     "grunt": "^0.4.5",
     "grunt-shell": "^1.1.1",
     "gruntfile-gtx": "^0.3.0",
+    "lodash": "^3.10.1",
```

Comment:
Can you include just lodash methods you require here rather than all of lodash.
(`lodash.pick` and `lodash.methods` are included below this)